src/components/Table/TableHeader/TableHeader.module.scss

```diff
@@ -1,12 +1,12 @@
 .tableHeader {
   display: grid;
   grid-template-columns: var(--table-grid-template-columns);
+  position: sticky;
+  top: 0;
```

Comment:
Is this always the case or only in certain scrolling logic (like full page scroll)?

Reply:
As far as I understand the header should always stick to top (we can opt-out with a prop in case a demand will arise)